flow360/component/simulation/services.py

```diff
@@ -96,10 +97,14 @@ def get_default_params(unit_system_name, length_unit) -> SimulationParams:
             reference_geometry=ReferenceGeometry(
                 area=1, moment_center=(0, 0, 0), moment_length=(1, 1, 1)
             ),
-            models=[Fluid()],
+            operating_condition=AerospaceCondition(velocity_magnitude=1),
```

Comment:
Personally I am against adding a default for operating condition. We should not try to describe the physical problem for the customer. 
We need to be careful to not accidentally ALWAYS trigger a case execution because we insert too many defaults.

Reply:
do you think we should require always: alpha, beta, density, temperature, material? Frontend is not ready and also does not hold any defaults. This way we can carry on default. Note that velocity_magnitude is removed from the dumped model.

Reply:
Let's discuss more details later. I do not understand the frontend statement.
We can merge this now. 
The defaults should be consistent with python client. I personally think very high level defaults should be as little as possible but once after the highest level we can provide as much defaults as possible. This allow user to describe the problem with minimal effort but still describe the problem.